### DISCUSSION TITLE
svelte 5.0 全新响应式API | liruifengv

### DISCUSSION THREAD

**Discussion by giscus** [liruifengv/liruifengv.com#30] [Announcements]
# svelte 5.0 全新响应式API | liruifengv

svelte发布了5.0版本，重构了响应式系统。

https://liruifengv.com/posts/svelte-5/

**Comment by Dustella** [comment]
它真的，越来越像 Vue3 Composition 了

**Reply by liruifengv** [reply]
哈哈哈哈哈。前端框架趋同了

**Comment by hongtaoh** [comment]
博客设计好棒！
